src/utils/scroll-to/scroll-to.ts

```diff
@@ -111,9 +111,10 @@ export class ScrollTo {
 
         // get element relative position from container
         if (target instanceof HTMLElement) {
-            const containerPosition: number = Math.round(container.offsetTop);
-            const elementPosition: number = Math.round(target.offsetTop);
-            targetLocation = Math.max((elementPosition - containerPosition) + offset, 0);
+            const scrollTop: number = (container.getBoundingClientRect().top * -1) + container.scrollTop;
+            targetLocation = (target.getBoundingClientRect().top + scrollTop) - container.clientTop;
+            targetLocation = targetLocation + offset;
+
```

Comment:
Est-ce que tu peux ajouter un cas de test dans le sandbox pour valider ce cas particulier ? Ça nous permettra de nous assurer que le fix fonctionne bien. Merci !

Reply:
@jipigi le cas est present en bas de la page du scroll-to